corehq/apps/app_manager/tasks.py

```diff
@@ -1,9 +1,23 @@
 from celery.task import task
 from corehq.apps.users.models import CommCareUser
+from corehq.util.decorators import serial_task
 
 
 @task(queue='background_queue', ignore_result=True)
 def create_user_cases(domain_name):
     from corehq.apps.callcenter.utils import sync_usercase
     for user in CommCareUser.by_domain(domain_name):
         sync_usercase(user)
+
+
+@serial_task('{app._id}', max_retries=0)
+def make_async_build(app):
+    latest_build = app.get_latest_app(released_only=False)
+    if latest_build.version == app.version:
+        return
+    errors = app.validate_app()
+    if not errors:
```

Comment:
yeah, i was just thinking more along the lines of someone not realizing that the update they just made on hq won't make it to mobile. if it's too big of a lift, probably no need to do it in this pr, but i think we would want to have some way to communicate to the user that the app won't build on the latest version when they're making updates in hq with errors